nearcache/nearcache_it_test.go

```diff
@@ -368,24 +369,36 @@ func TestNearCacheGet(t *testing.T) {
 
 func TestNearCacheInvalidateOnChange(t *testing.T) {
 	// port of: com.hazelcast.client.map.impl.nearcache.ClientMapNearCacheTest#testNearCacheInvalidateOnChange
-	tcx := newNearCacheMapTestContextWithExpiration(t, nearcache.InMemoryFormatObject, true)
-	tcx.Tester(func(tcx it.MapTestContext) {
-		m := tcx.M
-		t := tcx.T
-		ctx := context.Background()
-		const size = 118
-		populateServerMapWithString(ctx, tcx, size)
-		// populate Near Cache
-		populateNearCacheWithString(tcx, size)
-		oec := m.LocalMapStats().NearCacheStats.OwnedEntryCount
-		require.Equal(t, int64(size), oec)
-		// invalidate Near Cache from server side
-		populateServerMapWithString(ctx, tcx, size)
-		it.Eventually(t, func() bool {
-			oec := m.LocalMapStats().NearCacheStats.OwnedEntryCount
-			t.Logf("OEC: %d", oec)
-			return oec == 0
-		})
+	tcx := it.MapTestContext{T: t}
+	const port = 54001
+	clusterName := t.Name()
+	clsCfg := invalidationXMLConfig(clusterName, "non-existent", port)
+	tcx.Cluster = it.StartNewClusterWithConfig(1, clsCfg, port)
+	defer tcx.Cluster.Shutdown()
+	ctx := context.Background()
+	const size = 118
+	tcx.MapName = it.NewUniqueObjectName("map")
+	populateServerMapWithString(ctx, tcx, size)
```

Comment:
Since we need only name of the map, we can send it as a string instead of sending `MapTestContext`.